docs/get-started/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Get Started
+has_children: true
+nav_order: 1
+---
+
+# Get Started
+{: .no_toc }
+
+ONNX Runtime is a cross-platform accelerator for training and inference, compatible with PyTorch, TensorFlow/Keras, scikit-learn, and more.
+
+You can benefit from ONNX Runtime if you want to:
+
+* Improve inference performance of machine learning models
+* Reduce time and cost of training large models
+* Train in Python but deploy into a C#/C++/Java app
+* Run on different hardware and operating systems
+* Support models created in several different frameworks
+
+## Use ONNX Runtime for Inference
```

Comment:
split this into 2 pages, one for getting started with inference and one for getting started with training. then they will show up in the nav sidebar too.

Reply:
Done